deployability/modules/testing/tests/helpers/agent.py

```diff
@@ -0,0 +1,353 @@
+    def register_agent(inventory_path, manager_path):
+
+        with open(manager_path, 'r') as yaml_file:
+            manager_path = yaml.safe_load(yaml_file)
+        host = manager_path.get('ansible_host')
+
+        internal_ip = HostInformation.get_internal_ip_from_aws_dns(host) if 'amazonaws' in host else host
+
+        commands = [
+            f"sed -i 's/<address>MANAGER_IP<\/address>/<address>{internal_ip}<\/address>/g' {WAZUH_CONF}",
+            "systemctl restart wazuh-agent"
+        ]
+
+        Executor.execute_commands(inventory_path, commands)
+
+    @staticmethod
+    def uninstall_agent(inventory_path, wazuh_version=None, wazuh_revision=None) -> None:
+        os_type = HostInformation.get_os_type(inventory_path)
+        commands = []
+        if 'linux' in os_type:
+            distribution = HostInformation.get_linux_distribution(inventory_path)
+            os_name = HostInformation.get_os_name_from_inventory(inventory_path)
+            if os_name == 'opensuse' or os_name == 'suse':
+                    commands.extend([
+                        "zypper remove --no-confirm wazuh-agent",
+                        "rm -r /var/ossec"
+                    ])
+            else:
+                if distribution == 'deb':
+                        commands.extend([
+                            "apt-get remove --purge wazuh-agent -y"
+
+                        ])
+                elif distribution == 'rpm':
+                    commands.extend([
+                        "yum remove wazuh-agent -y",
+                        f"rm -rf {WAZUH_ROOT}"
+                    ])
+
+
+            system_commands = [
+                    "systemctl disable wazuh-agent",
+                    "systemctl daemon-reload"
+            ]
+
+            commands.extend(system_commands)
+        elif 'windows' in os_type:
+            commands.extend([
+                f"msiexec.exe /x wazuh-agent-{wazuh_version}-1.msi /qn"
+            ])
+        elif 'macos' in os_type:
+            commands.extend([
+                "/Library/Ossec/bin/wazuh-control stop",
+                "/bin/rm -r /Library/Ossec",
+                "/bin/launchctl unload /Library/LaunchDaemons/com.wazuh.agent.plist",
+                "/bin/rm -f /Library/LaunchDaemons/com.wazuh.agent.plist",
+                "/bin/rm -rf /Library/StartupItems/WAZUH",
+                "/usr/bin/dscl . -delete '/Users/wazuh'",
+                "/usr/bin/dscl . -delete '/Groups/wazuh'",
+                "/usr/sbin/pkgutil --forget com.wazuh.pkg.wazuh-agent"
+            ])
+
+        Executor.execute_commands(inventory_path, commands)
+
+
+    @staticmethod
+    def uninstall_agents( inventories_paths=[], wazuh_version: Optional[List[str]]=None, wazuh_revision: Optional[List[str]]=None) -> None:
+        for index, inventory_path in enumerate(inventories_paths):
+            WazuhAgent.uninstall_agent(inventory_path, wazuh_version[index], wazuh_revision[index])
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Too much blank lines between functions
```suggestion
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4998/commits/d05be9a41f9aa224b48f5fb8208a042f2e9aeb7f